doc/migration/version-upgrade-guide.md

```diff
@@ -972,6 +972,22 @@ With:
 ```yaml 
 - name: {{your operator name which is the value of metadata.name in this file}}
 ```
+### Notable Changes
```

Comment:
Here we need to add it before [Remove Ansible container sidecar](https://github.com/operator-framework/operator-sdk/blob/e113b6c3caf643a192f172c6bec60922c601e977/doc/migration/version-upgrade-guide.md#remove-ansible-container-sidecar) in the Breaking change section as follows. 

--- 

### Breaking changes

#### In the test-framwork

The methods `ctx.GetOperatorNamespace()` and `ctx.GetWatchNamespace()` was added `pkg/test` in order to replace `ctx.GetNamespace()` which is  deprecated. In this way replace the use of `ctx.GetNamespace()` in your project with `ctx.GetOperatorNamespace()`

#### Remove Ansible container sidecar
... 
----

Then, regards the commands we need to another section after the Breaking Change above as follows. See the example: https://github.com/operator-framework/operator-sdk/blob/master/doc/migration/version-upgrade-guide.md#breaking-changes-on-commands

### Breaking Changes on Commands 

This release contains breaking changes in some commands.

- The `--namespace` flag from `operator-sdk run --local` and `operator-sdk test --local` command was deprecated and was replaced by `--watch-namespace` and `--operator-namespace` .  

Note that `--watch-namespace` can be used to set the namespace(s) which the operator will watch for changes. It will set the environment variable `WATCH_NAMESPACE`.  Use explicitly an empty string to watch all namespaces or inform a List of namespaces such as "ns1,ns2" when the operator is [cluster-scoped](https://github.com/operator-framework/operator-sdk/blob/master/doc/operator-scope.md#cluster-scoped-operator-usage). If you use a List, then it needs contains the namespace where the operator is "deployed" in since the default metrics implementation will manage resources in the Operator's namespace. By default, it will be the Operator Namespace. 

Then, use the flag `--operator-namespace` to inform the namespace where the Operator will be "deployed" in and then, it will set the environment variable `OPERATOR_NAMESPACE`. If this value is not set, then it will be the namespace defined as default in the Kubeconfig. 
---

Reply:
done